python/ray/dataframe/test/test_io.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+import numpy as np
+import pandas as pd
+import ray
+import ray.dataframe as rdf
+import ray.dataframe.io as io
+import os
+
+TEST_PARQUET_FILENAME = 'test.parquet'
+TEST_CSV_FILENAME = 'test.csv'
+SMALL_ROW_SIZE = 2000
+LARGE_ROW_SIZE = 7e6
+
+
+@pytest.fixture
+def ray_df_equals_pandas(ray_df, pandas_df):
+    return rdf.to_pandas(ray_df).sort_index().equals(pandas_df.sort_index())
```

Comment:
Ideally eventually I think we should remove the `sort_index()` because we want order to be compatible too. I think I have fixed most of this, so I will test that.